packages/core/admin/admin/src/content-manager/components/DynamicTable/CellContent/RelationMultiple/tests/index.test.js

```diff
@@ -23,14 +23,14 @@ jest.spyOn(axiosInstance, 'get').mockResolvedValue({
 });
 
 const DEFAULT_PROPS_FIXTURE = {
+  contentType: {
+    uid: 'api::address.address',
+  },
   fieldSchema: {
     type: 'relation',
     relation: 'manyToMany',
     target: 'api::category.category',
   },
-  queryInfos: {
```

Comment:
Are you moving the request earlier in the tree? How do you handle it with i18n then?

Reply:
Answered below.